packages/website/md/docs/smart_contracts/2.0.0/introduction.md

```diff
@@ -0,0 +1,8 @@
+Welcome to the [0x smart contracts](https://github.com/0xProject/contracts) documentation! This documentation is intended for dApp developers who want to integrate 0x exchange functionality directly into their own smart contracts.
+
+### Helpful wiki articles:
+
+*   [Overview of 0x protocol architecture](https://0xproject.com/wiki#Architecture)
+*   [0x smart contract interactions](https://0xproject.com/wiki#Contract-Interactions)
+*   [Deployed smart contract addresses](https://0xproject.com/wiki#Deployed-Addresses)
+*   [0x protocol message format](https://0xproject.com/wiki#Message-Format)
```

Comment:
So for V2, only the `Deployed smart contract addresses` wiki article remains. Let's redirect them to: https://github.com/0xProject/0x-protocol-specification/blob/master/v2/v2-specification.md

Reply:
We also need to remove all these links from the V1 introduction.md